frontend/src/v5/store/federations/federations.types.ts

```diff
@@ -14,8 +14,6 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Action } from 'redux';
-
 export interface IFederation {
```

Comment:
Is there a logic behind where we use the "I" names? (e.g. *IFederationsState* instead of just *FederationsState*). I understand that these are interfaces and not types, but in this same file there is the interface *SurveyPoint*. Should this be *ISurveyPoint*?

Could it also be worth using the "I" names for types as well? Just so it's clear that FederationStats is a type and not an object containing federation stats? I suppose in the context it's used it should always be quite clear, but it's something to consider

Reply:
I changes SurveyPoint to ISurveyPoint to match the style with interfaces.  I think I prefer without the I in general, but if I remember correctly there was a rule to enforce the I in interfaces.